src/ifc5d/ifc5d/ifc5Dspreadsheet.py

```diff
@@ -101,6 +104,18 @@ def get_cost_item_values(cost_item: Union[ifcopenshell.entity_instance, None]) -
             )
         return values
 
+    @staticmethod
+    def cost_item_is_a_sum(cost_item: ifcopenshell.entity_instance) -> bool:
+        cost_values = []
+        if cost_item.is_a("IfcCostItem"):
+            cost_values = cost_item.CostValues
+        elif cost_item.is_a("IfcCostValue"):
```

Comment:
@carlopav maybe i'm missing something, but i don't understand this: can this function be used also with non cost item as input (in this case, cost value)? If i understand it correctly, this function checks if a cost item is a sum or not, based on the category condition. Is it possible that this function is used with cost values as input?

Reply:
well... i copied it from ifc2json.py, row 83. I thought it was perhaps something useful to keep both possibilities so I didn't get rid of it.

Reply:
My opinion is to remove that if it is not necessary, but you can just leave as it is if you want :-)

Reply:
Good catch, agree. it can give a false impression that code that's using it can somehow provide non-ifccostitems as inputs.

Reply:
Done :)

Reply:
Sure :) My bad!